prisma/schema.prisma

```diff
@@ -16,9 +16,11 @@ model User {
   username String @unique
 
   devices Device[]
+  cats    Cat[]
 
   updated_at DateTime @updatedAt
   created_at DateTime @default(now())
+  deleted_at DateTime?
```

Comment:
Is this GDPR compliant? I don't think it is.
What's the reason for adding soft deletion?

Reply:
Sorry - I stole this code from one of my previous repo's to fix an issue with relation deletions. The reason my previous repo does it is to establish and defend legal claims against explicit content - It isn't needed here so I will switch